zipkin-lens/src/components/Dependencies/DependenciesGraph.js

```diff
@@ -71,8 +79,8 @@ class DependenciesGraph extends React.Component {
             name: 'dependencies-graph',
             updated: new Date().getTime(),
             maxVolume: maxVolume * 2000,
-            nodes: graph.allNodes(),
-            connections: graph.allEdges(),
+            nodes,
+            connections
           }}
           objectHighlighted={this.handleObjectHighlighted}
           match={filter}
```

Comment:
`match` prop is no longer needed.
Would you remove this line?

Reply:
Good call 👍